metadata.json

```diff
@@ -1,16 +1,15 @@
 {
-  "_generated": "Generated by SweetTooth, do not edit",
-  "description": "Bring Out Submenu Of Power Off/Logout Button",
-  "name": "Bring Out Submenu Of Power Off/Logout Button",
-  "settings-schema": "org.gnome.shell.extensions.brngout",
-  "shell-version": [
-    "45"
-  ],
-  "url": "https://github.com/PRATAP-KUMAR/Bring-Out-Submenu-of-Power-Off-Logout",
-  "uuid": "BringOutSubmenuOfPowerOffLogoutButton@pratap.fastmail.fm",
-  "version": 46,
-  "donations": {
-    "buymeacoffee": "pratappanabaka",
-    "paypal": "pratappanabaka"
-  }
-}
\ No newline at end of file
+    "description": "Bring Out Submenu Of Power Off/Logout Button",
+    "name": "Bring Out Submenu Of Power Off/Logout Button",
+    "settings-schema": "org.gnome.shell.extensions.bring-out-submenu-of-power-off-logout",
+    "shell-version": [
+        "45"
+    ],
+    "url": "https://github.com/PRATAP-KUMAR/Bring-Out-Submenu-of-Power-Off-Logout",
+    "uuid": "bring-out-submenu-of-power-off-logout@pratap.fastmail.fm",
```

Comment:
Sorry this cant be changed, because this extension will be treated as a new one by extensions.gnome.org

Reply:
you can automate this in your makefile, I mean changing the `uuid`.

Thanks